### PR TITLE
add photo Bandit

### PR DESCRIPTION
**Pull request by AndreyNemchenko**

### CODE REVIEW

**Review by suchov — APPROVED** [review]

**Comment by artem-trubin** [comment]
Welcome, well done!

**Comment by artem-trubin** [comment]
Oh, wait, the link is broken. It seems that you're linking an absent file. Did you forget to add a file?

File: README.md [hunk 1]
```diff
@@ -4049,5 +4049,9 @@ Does anyone loves archery and choppers the same as I do? :)
 My name is [Kostiantyn](https://github.com/rukhlovka)! And this is my cat and my hands when I was learning Git.
 ![Kitten](./assets/images/cat-Mini.jpg)
 
+
+## Hi! i'm Andre! 
+![Screenshot-image-link](./assets/images/Bandit.jpg)
```
Comment (suchov) [inline README.md:4054]: @artem-trubin noticed on time; it seems like the link is broken.
Have you forgotten to add the file?

**Review by suchov — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (AndreyNemchenko) [reply README.md:4087]: @suchov Hi, I fixed the problem. Please, review.

File: README.md [hunk 2]
```diff
@@ -4082,8 +4082,13 @@ Does anyone loves archery and choppers the same as I do? :)
 My name is [Kostiantyn](https://github.com/rukhlovka)! And this is my cat and my hands when I was learning Git.
 ![Kitten](./assets/images/cat-Mini.jpg)
 
+
+## Hi! i'm Andre! 
+![Screenshot-image-link](./assets/images/Bandit.jpg) 
+
 ## Hello, Kottans
 My name is Alex.
+<<<<<<< HEAD
```
Comment (suchov) [inline README.md:4091]: Great work.
The only thing is that this line "<<<<<<< HEAD" should be deleted - it's a part of a "conflict" message.

**Review by suchov — CHANGES_REQUESTED** [review]

**Comment by AndreyNemchenko** [comment]
@suchov I fixed!

**Review by suchov — APPROVED** [review]

**Comment by suchov** [comment]
Looks great, thx!

**Pull request merged**